File: Rmm.Domain/Rmm.Domain.Core/Models/DeviceState.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using Rmm.Domain.DAL.Enums;

namespace Rmm.Domain.Core.Models
{
    public class DeviceState: IDevicePartialData
    {
        public Guid SourceDeviceId { get; set; }
        public DeviceStatus Status { get; set; }
        public int RunTimeS { get; set; }
        public double LastAnswerTime { get; set; }
    }
}

File: Rmm.WebClient/src/app/devicePanel-module/models/deviceState.ts
import { DeviceStatus } from '../enums/deviceStatus';
import { DeviceStateResponse } from './response/deviceState-response';

export class DeviceState {
    public sourceDeviceId!: string;
    public status?: DeviceStatus;
    public runTimeS?: number;
    public lastAnswerTime?: number;
    
    constructor(resp: DeviceStateResponse) {
        this.sourceDeviceId = resp.sourceDeviceId;
        this.status = resp.status;
        this.runTimeS = resp.runTimeS;
        this.lastAnswerTime = resp.lastAnswerTime;
    }
}